DevEdu.API/DevEdu.API/Controllers/LessonController.cs

```diff
@@ -158,20 +158,22 @@ public void AddTopicToLesson(int lessonId, int topicId)
         // api/lesson/{lessonId}/user/{userId}
         [HttpPost("{lessonId}/user/{userId}")]
         [Description("Adds student to lesson")]
-        [ProducesResponseType(StatusCodes.Status204NoContent)]
-        public void AddStudentToLesson(int lessonId, int userId )
+        [ProducesResponseType(typeof(StudentLessonShortOutputModel), StatusCodes.Status204NoContent)]
+        public StudentLessonShortOutputModel AddStudentToLesson(int lessonId, int userId )
```

Comment:
please remove the "StudentLessonShortOutputModel" class and use StudentLessonOutputModel
also, in StudentLessonOutputModel you've got to change "public int UserId" to "public UserInfoOutputModel User" and add all the needed logic for it